### PR TITLE
[dist] Add OBS API Setup for systemd

### PR DESCRIPTION
**Pull request by leemgs**
Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

**Comment by leemgs** [comment]
This PR is to fix https://github.com/openSUSE/open-build-service/issues/2997.

**Comment by bgeuken** [comment]
LGTM

**Comment by mschnitzer** [comment]
@M0ses I assigned you as I heard you are somehow involved in that. ( as a reminder ;-) )

File: dist/systemd/obsapisetup.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Description=OBS API Setup
+After=network.target
+         
+[Service]
+EnvironmentFile=/etc/sysconfig/obs-server
+Type=oneshot
+ExecStart=/usr/lib/obs/server/setup-appliance.sh --non-interactive --setup-only
+ExecReload=/bin/kill -HUP $MAINPID
```
Comment (M0ses) [inline dist/systemd/obsapisetup.service]: What is the intention behind an reload of setup-appliance.sh? I´m quite sure that we do not trap SIGHUP.

**Pull request closed without merging**